e3db/client.py

```diff
@@ -902,13 +922,46 @@ def search(self, query):
         return qr
 
     def __search(self, query):
+        """
+        Private Method to send search request to the search service and return a json response.
+        
+        Parameters
+        ----------
+        query: Search
+            Search object represents the query made to the search service.
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"